### PR TITLE
Why do you download the entire dataset every 4.5 hours when it's only updated every 5 days??

### PR DESCRIPTION
**Pull request by shade**
Well, I don't think anyone wants to waste 1gb of bandwidth every couple hours on redundant course data. This just checks the github api to see if there was a new commit made to the datasets repo. I also changed the the cron job to execute every 1.5 hours

### CODE REVIEW

File: src/db/index.js [hunk 1]
```diff
@@ -52,6 +54,39 @@ db.sync = () => {
   db.update('courses')
 }
 
+db.check = (callback) => {
```
Comment (arkon) [inline src/db/index.js:57]: A promise would be nice here.

File: src/db/index.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ import childProcess from 'child_process'
 import mongoose from 'mongoose'
 
 let db = {}
+//I'm not sure why you guys use let
```
Comment (qasim) [inline src/db/index.js:9]: We use `let` statements as part of the new ES2015 syntax, more details on `let` vs `var` (and `const`) here: https://strongloop.com/strongblog/es6-variable-declarations/

**Comment by qasim** [comment]
Your implementation is definitely smarter. We hadn't settled on the speed of scraping / retrieving yet but this definitely offers a nice way of not deciding at all. Thanks!

Just a side-note, we follow the [feross/standard](https://github.com/feross/standard) JS style guide. I'll add comments for the lines that need to be modified. You can run `$ npm run lint` to check if your branch follows the style guide (it should return nothing if all is good, otherwise you'll get a descriptive list of errors).

**Comment by shade** [comment]
Yea, my bad, I don't usually  use spaces and I couldn't tell where you were using tabs or spaces.

**Comment by qasim** [comment]
It's all good! Once you fix them I'll merge. :grin:

File: src/db/index.js (hunk 1 above)
Comment (qasim) [inline src/db/index.js:57]: We'll probably tackle all callback patterns in one swoop sometime in the future. As of now, it's fine.

**Comment by qasim** [comment]
Merged this to #51 .

**Pull request closed without merging**